ui/src/shared/components/FancyScrollbar.js

```diff
@@ -9,34 +9,49 @@ class FancyScrollbar extends Component {
 
   static defaultProps = {
     autoHide: true,
+    autoHeight: false,
   }
 
   render() {
-    const {autoHide, children, className} = this.props
+    const {autoHide, autoHeight, children, className, maxHeight} = this.props
 
     return (
       <Scrollbars
-        className={classnames('fancy-scroll--container', {[className]: className})}
+        className={classnames('fancy-scroll--container', {
+          [className]: className,
+        })}
         autoHide={autoHide}
         autoHideTimeout={1000}
         autoHideDuration={250}
-        renderTrackHorizontal={props => <div {...props} className="fancy-scroll--track-h"/>}
-        renderTrackVertical={props => <div {...props} className="fancy-scroll--track-v"/>}
-        renderThumbHorizontal={props => <div {...props} className="fancy-scroll--thumb-h"/>}
-        renderThumbVertical={props => <div {...props} className="fancy-scroll--thumb-v"/>}
+        autoHeight={autoHeight}
+        autoHeightMax={maxHeight ? maxHeight : null}
```

Comment:
When is `maxHeight` falsey?  If it's null? Then you can do:
`autoHeightMax={maxHeight}`?